clients/web/src/dialog.js

```diff
@@ -70,24 +75,50 @@ function confirm(params) {
         yesText: 'Yes',
         yesClass: 'btn-danger',
         noText: 'Cancel',
-        escapedHtml: false
+        escapedHtml: false,
+        msgConfirmation: false,
+        additionalText: '',
+        name: ''
     }, params);
     $('#g-dialog-container').html(ConfirmDialogTemplate({
         params: params
     })).girderModal(false).one('hidden.bs.modal', function () {
         $('#g-confirm-button').off('click');
     });
 
-    var el = $('#g-dialog-container').find('.modal-body>p');
+    const el = $('#g-dialog-container').find('.modal-body>p:first-child');
     if (params.escapedHtml) {
         el.html(params.text);
     } else {
         el.text(params.text);
     }
+    if (params['msgConfirmation']) {
+        $('#g-confirm-text').css({'max-width': '50%'});
```

Comment:
It would be better to have this in a `styl` definition file.  We have other dialog style css in `clients/web/src/stylesheets/layout/global.styl`.

Reply:
Yes you're right!
Do you think this should use a class instead and put it within global.styl ?
Even if it applies only on this special case

Reply:
I would leave it id specific, leaving it to be generalized if we need to at a later time.

Reply:
Ok sounds great!